tree/treeplayer/src/TTreeProcessorMT.cxx

```diff
@@ -43,17 +43,22 @@ struct EntryCluster {
 
 ////////////////////////////////////////////////////////////////////////////////
 /// Construct fChain, also adding friends if needed and injecting knowledge of offsets if available.
-void TTreeView::MakeChain(const std::string &treeName, const std::vector<std::string> &fileNames,
+/// \param[in] treeNames Name of the tree for each file in `fileNames`.
+/// \param[in] fileNames Files to be opened.
+/// \param[in] friendInfo xxx No idea what this is.
+/// \param[in] nEntries Number of entries to be processed.
+/// \param[in] friendEntries xxx Why do we need that? 
```

Comment:
Ah, well. The github suggestions don't really work so well if the suggestion is incomplete ...

If you look closely, you see that I had no clue what `friendInfo` and `friendEntries` are supposed to do. That's why I started the docstring.

Reply:
oh boy xD

Reply:
fixed